huggingface/pytorch/training/docker/1.9/py3/cu111/Dockerfile.trcomp.gpu

```diff
@@ -0,0 +1,91 @@
+# https://github.com/aws/deep-learning-containers/blob/master/available_images.md 
+# refer to the above page to pull latest Pytorch image
+
+# docker image region us-west-2
+FROM 763104351884.dkr.ecr.us-west-2.amazonaws.com/pytorch-training:1.9-gpu-py38-cu111-ubuntu20.04-v1
```

Comment:
I believe this image is 1.9.1 as well. That was the latest Base DLC release.

Reply:
I believe that you are uninstalling the underlying binary from this image and re-installing a new one right? Thus the version needs to correspond to the pytorch version that is ultimately installed in the container